airflow/models/skipmixin.py

```diff
@@ -61,7 +61,7 @@ def _set_state_to_skipped(self, dag_run, execution_date, tasks, session):
             if execution_date is None:
                 raise ValueError("Execution date is None and no dag run")
 
-            self.log.warning("No DAG RUN present this should not happen")
+            self.log.debug("No DAG run present. This should not happen.")
```

Comment:
As a user I would have no idea what this log may mean

Reply:
But it's a sign that _something_went wrong - having this at debug feels pointless

Reply:
I agree, we should provide more useful information "This should not happen"